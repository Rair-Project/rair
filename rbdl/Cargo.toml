[package]
name = "rbdl"
version = "0.1.0"
authors = ["oddcoder <ahmedsoliman@oddcoder.com>"]
edition = "2018"
autotests = false

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
[lib]
name = "rbdl"
proc-macro = true

[dependencies]
syn = {version = "1.0", features=["extra-traits", "full"]}
quote = "1.0"
[[test]]
name = "rbdl_testing"
path = "tests/run_tests.rs"

[dev-dependencies]
trybuild = "1.0"