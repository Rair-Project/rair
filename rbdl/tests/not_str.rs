use rbdl::rbdl_file;

rbdl_file!(2 + 2);

fn main() {

}