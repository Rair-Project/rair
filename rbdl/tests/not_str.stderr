error: expected string literal
 --> $DIR/not_str.rs:3:12
  |
3 | rbdl_file!(2 + 2);
  |            ^
